exercises/practice/isogram/.meta/example.sml

```diff
@@ -0,0 +1,17 @@
+val isIsogram: string -> bool =
+  let
+    val a = Char.ord #"a"
+    fun letterValue (c: char): word =
+      Word.<<(0wx1, Word.fromInt(Char.ord (Char.toLower c) - a))
+    fun recurse (seen: word) (l: char list): bool =
```

Comment:
Might be helpful to add a short docstring to `recurse`, in particular explaining the purpose of `seen` from which alot of the other logic follows (IMO). An example might also help, like "The letter `c` is encoded as `...00100`".

Reply:
Comments added.